datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts

```diff
@@ -16,6 +16,24 @@ function matchesTagsOrTermsOrDescription(field: SchemaField, filterText: string,
     );
 }
 
+function matchesBusinessAttributesProperties(field: SchemaField, filterText: string, entityRegistry: EntityRegistry) {
```

Comment:
nice! could we add some tests in `filterSchemaRows.test.ts` for this new functionality?

Reply:
sure I will add that